src/components/MediaObject/MediaObject.md

```diff
@@ -102,6 +102,64 @@ const avatar = (size) => (
 </div>
 ```
 
+With ellipsis:
+
+```js { "props": { "data-example": "with ellipsis" } }
```

Comment:
if we change `ellipsis` to `allowTextWrap` then we should also update this test name to "with allow text wrap"